commands/build.go

```diff
@@ -597,11 +619,6 @@ func buildCmd(dockerCli command.Cli, rootOpts *rootOptions, debugConfig *debug.D
 
 	flags.StringArrayVar(&options.platforms, "platform", platformsDefault, "Set target platform for build")
 
-	if confutil.IsExperimental() {
-		flags.StringVar(&options.printFunc, "print", "", "Print result of information request (e.g., outline, targets)")
-		cobrautil.MarkFlagsExperimental(flags, "print")
-	}
```

Comment:
Compared to https://github.com/docker/buildx/pull/2494/files#diff-9a2da9eceb368adfae46838969a2274770a558a410fa374707616b0c5737d9b6R21, I just remove this flag as it was under experimental mode anyway.